ledger/sandbox-on-x/BUILD.bazel

```diff
@@ -339,6 +341,53 @@ da_scala_binary(
     ],
 )
 
+# An app that outputs the default config
+da_scala_binary(
+    name = "generate-default-config-app",
+    main_class = "com.daml.ledger.sandbox.DefaultConfigGenApp",
```

Comment:
Isn't running sandbox-on-x with a `convert-config` command achieve exactly the same?

Reply:
`convert-config` might work but I haven't been checking that.
Its core purpose is to convert from the legacy config format which seems misaligned from wanting to render the non-legacy default config

Reply:
This mode takes the default and modifies it using legacy flags. AFAICT if you provide empty legacy config (i.e. no flags) it will spit out the default

Reply:
tried it, it is not exactly how I said it, you need to do:
```
bazel run //ledger/sandbox-on-x:app -- convert-config --participant participant-id=default,port=6861
```

Reply:
I agree that using the above is brittle and not ideal.
I would argue though that adding a startup mode would be better than another main class

Reply:
Ok, will add a startup mode